x-pack/plugins/security_solution/public/detections/containers/detection_engine/rules/use_preview_rule.ts

```diff
@@ -71,20 +71,25 @@ export const usePreviewRule = ({
     [advancedOptions]
   );
 
+  let isInvalidInterval = false;
   if (advancedOptions) {
     const timeframeDuration =
       (advancedOptions.timeframeEnd.valueOf() / 1000 -
         advancedOptions.timeframeStart.valueOf() / 1000) *
       1000;
 
     const { unit: intervalUnit, value: intervalValue } = getTimeTypeValue(advancedOptions.interval);
-    const { unit: lookbackUnit, value: lookbackValue } = getTimeTypeValue(advancedOptions.lookback);
     const duration = moment.duration(intervalValue, intervalUnit as 's' | 'm' | 'h');
-    duration.add(lookbackValue, lookbackUnit as 's' | 'm' | 'h');
     const ruleIntervalDuration = duration.asMilliseconds();
+    if (ruleIntervalDuration > timeframeDuration) {
+      isInvalidInterval = true;
+    }
```

Comment:
It would be great to add a comment on why we consider the interval invalid in this case + include a link to https://github.com/elastic/kibana/issues/137422.